modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -314,13 +317,23 @@ function FieldBase({
 							>
 								{label && showLabel && label}
 
-								<FieldProperties
-									required={required}
+								{required && <RequiredProperty />}
```

Comment:
Break line after this one

Reply:
Done. See it here https://github.com/liferay-forms/liferay-portal/pull/484/commits/4a319cce54f917cfecbd62c1bda54262e93cc6b2